packages/hardhat/contracts/periphery/ControllerHelper.sol

```diff
@@ -700,6 +700,41 @@ contract ControllerHelper is UniswapControllerHelper, EulerControllerHelper, IER
                         uint8(ControllerHelperDataType.CALLBACK_SOURCE.GENERAL_SWAP),
                         ""
                     );
+                } else if (
+                    data[i].rebalanceVaultNftType == ControllerHelperDataType.RebalanceVaultNftType.CollectFees
+                ) {
+                    ControllerHelperDataType.CollectParams memory collectParams = abi.decode(
```

Comment:
Can u rename:
- `ControllerHelperDataType.CollectParams` => `ControllerHelperDataType.CollectFeesParams`
- `memory collectParams` => `memory collectFeesParams`

just because the `RebalanceVaultNftType` called `CollectFees`